addons/point_of_sale/tests/test_pos_products_with_tax.py

```diff
@@ -294,6 +296,98 @@ def test_return_order(self):
         for t1, t2 in zip(sorted(manually_calculated_taxes), sorted(tax_lines.mapped('balance'))):
             self.assertAlmostEqual(t1, t2, msg='Taxes should be correctly combined and should be debit.')
 
+    def test_pos_create_entry_move_tags_with_pos(self):
+        """ Test normal entry move creation with opened POS session
+            Check if move tags are correctly created
+        """
+        # Create a new tax with its corresponding tax report lines
+        # in order to simulate the tags affectation
+        tax_report = self.env["account.tax.report"].create({
+            "name": "Tax report",
+        })
+        base_20 = self.env["account.tax.report.line"].create({
+            "name": "Base 20",
+            "tag_name": "20B",
+            "report_id": tax_report.id,
+            "sequence": 1,
+        })
+        base_20_tag_plus, base_20_tag_minus = base_20.tag_ids.sorted("tax_negate")
+        tax_20 = self.env["account.tax.report.line"].create({
+            "name": "20",
+            "tag_name": "20T",
+            "report_id": tax_report.id,
+            "sequence": 2,
+        })
+        tax_20_tag_plus, tax_20_tag_minus = tax_20.tag_ids.sorted("tax_negate")
+        tax_20_incl = self.env['account.tax'].create({
+            "sequence": 20,
+            "description": "20%",
+            "name": "20%",
+            "amount": 20,
+            "amount_type": "percent",
+            "type_tax_use": "sale",
+            'price_include': True,
+            "invoice_repartition_line_ids": [
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'tag_ids': [(6, 0, base_20_tag_plus.ids)],
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                    'account_id': self.tax_received_account.id,
+                    'tag_ids': [(6, 0, tax_20_tag_plus.ids)],
+                }),
+            ]
+        })
+        self.open_new_session()
+        self.env['pos.order'].create_from_ui([self.create_ui_order_data([])])
```

Comment:
I know it is working but could you set at least one product for the order?
It's better to have something that is a little bit realistic.
If someone rework this api and break it with empty list, (s)he will have to modify your test to make it work.

Reply:
> It's the contrary of your last review sweat_smile

Nope, if you look at it, in the code example I give, I have left one product:
https://github.com/odoo/odoo/pull/90943#discussion_r869217800